apps/docs/errors/unkey/data/ratelimit_namespace_gone.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Ratelimit Namespace Gone
+---
+
+## Description
+
+This error occurs when you attempt to use a ratelimit namespace that has been deleted. Once a namespace is deleted, it cannot be restored through the API or dashboard.
+
+## Error Code
+
+`unkey/data/ratelimit_namespace_gone`
+
+## HTTP Status Code
+
+`410 Gone`
+
+## Cause
+
+The ratelimit namespace you're trying to access was previously deleted and is no longer available through the API or dashboard.
+
+## Resolution
+
+<Note>
+Contact [support@unkey.com](mailto:support@unkey.com) with your workspace ID and namespace name if you need this namespace restored.
```

Comment:
either we change this to unkey.dev or we add support@unkey in plain

I can't remember why we don't have that yet, cc @perkinsjr